integration_tests/e2e/workAndSkillsPage.cy.ts

```diff
@@ -132,44 +132,22 @@ context('Work and skills page', () => {
       })
 
       context('Work and activities card', () => {
-        it('The card is displayed', () => {
-          const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.WAA_card().should('exist')
-        })
-
-        it('The card summary header contains Work and activities', () => {
-          const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.WAA_header().contains('Work and activities')
-        })
-
-        it('The card summary header contains Work and activities', () => {
-          const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.WAA_header().contains('Work and activities')
-        })
-
         it('The card has a list key that should contain Braille am', () => {
           const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.WAA_keyChild1().contains('Braille am')
-        })
-
-        it('The card contains the text Unacceptable absences', () => {
-          const workAndSkillsPage = Page.verifyOnPage(WorkAndSkillsPage)
-          workAndSkillsPage.WAA_absences().contains('Unacceptable absences')
+          workAndSkillsPage.workAndActivities().currentActivities(0).activity().contains('Braille am')
         })
 
-        it('The card contains a key with the text Last 30 days', () => {
+        it('Displays the number of absenses in the last 30 days', () => {
```

Comment:
```suggestion
        it('Displays the number of absences in the last 30 days', () => {
```
spotted a typo - sorry!

Reply:
I can never spell it :') I'll fix this in a next PR